api/users_test.go

```diff
@@ -56,6 +56,28 @@ func (m *storeMock) Update(_ context.Context, user *model.User) (*model.User, er
 	return args.Get(0).(*model.User), args.Error(1)
 }
 
+func (m *storeMock) FindStockBySymbol(_ context.Context, symbol string) (*model.Stock, error) {
+	args := m.Called(symbol)
+	if stock, ok := args.Get(0).(*model.Stock); ok {
+		return stock, args.Error(1)
+	}
+
+	return nil, args.Error(1)
+}
```

Comment:
```suggestion
func (m *storeMock) FindStockBySymbol(_ context.Context, symbol string) (*model.Stock, error) {
	args := m.Called(symbol)
	if args.Get(0) == nil {
		return nil, args.Error(1)
	}
	return args.Get(0).(*model.Stock), args.Error(1)
}
```

Since you write the tests, no need to check the type, just make sure it's not nil